lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -2262,7 +2337,14 @@ namespace {
       conformance->setInvalid();
       return conformance;
     }
-
+    
+    // If this is the DynamicMemberLookupProtocol, manually check its
+    // requirement.
+    if (Proto->isSpecificProtocol(
```

Comment:
There are many other places where we directly call ModuleDecl::lookupConformance() and they will miss this check, but I'm not sure if it matters

Reply:
I think it is fine.  Whatever defines a conformance will get checked.